### ISSUE TITLE
Change the Optimization Method (AdamW --> diffGrad)

### ISSUE THREAD

**Issue by cepLI** [RVC-Project/Retrieval-based-Voice-Conversion-WebUI#952]
Adamw was replaced by a new optimization method - **diffgrad**. What do you think about it? Will you add it to increase the accuracy in the experiment?

```
    optim_g = torch.optim.AdamW(
        net_g.parameters(),
        hps.train.learning_rate,
        betas=hps.train.betas,
        eps=hps.train.eps,
    )
    optim_d = torch.optim.AdamW(
        net_d.parameters(),
        hps.train.learning_rate,
        betas=hps.train.betas,
        eps=hps.train.eps,
    )
```

**Source diffGrad:** https://github.com/shivram1987/diffGrad

**Comment by HSultankhan** [comment]
@jhchase  In which file are you changing this? Can you guide please?
